CHANGELOG.md

```diff
@@ -2,6 +2,172 @@
+* Toolbars
+  * Added new "Export as PDF" toolbar item (#3508). It and the "Print" toolbar
+    item have also been added to the default layouts.
+* Layers
+  * Added a context menu for the layer sidebar preview (#3204).
+  * Added a feature for merging layers down. This can be accessed via the
+    `Journal`menu or the new layer preview context menu (#2784, #3204).
+  * Added `Edit > Move Layer Up/Down` menu items for moving layers up/down
+    (#4352).
+  * Added a CLI flag to choose which layer(s) to export (#4143).
+* Plugins
+  * Added new plugin APIs, including export options, getting page info, adding
+    strokes etc. (#3677, #3566, #3688, #4359, #4397).
+  * Added a new plugin "Export" used to export the current document to PDF, SVG,
+    or PNG format with `<Shift><Alt>p/s/n`, respectively (#3566).
+  * Added a new plugin "HighlightPosition" used to toggle the cursor highlight
+    (#3023).
+  * Added support for additional plugin loading paths (#1155, #2422). Notably,
+    plugins will also be searched for in the `plugins` folder contained in the
+    user's config directory (see the "File Locations" section of the
+    website/guide for where to find this folder).
+  * QuickScreenshot plugin: added `maim` and `xclip` program support (#3718).
+* Audio recordings
+  * All elements without audio now become faded when the Play Object tool is
+    used (#2874).
+  * Window: fixed issues with saving/loading audio files with unicode characters
+    in their names (#3783).
+* Vertical Spacing Tool
+  * Added capability to move elements above the line by holding Ctrl (#3334).
+  * Improved performance (#3334, #4306).
+* TeX Tool
+  * Reimplemented the LaTeX editor using the gtksourceview library (#2809),
+    enabling multiline input, syntax highlighting, undo/recode, word wrap, and a
+    resizable code editor.
+  * Added a tab for viewing the output of the TeX command (#2809).
+  * Added Preferences settings for changing the syntax highlighting theme, word
+    wrap settings, font, default formula, and other settings (#2809, #4264).
+  * Changed the TeX background color to become dark when a light font color is
+    selected (#2809).
+  * Fixed a bug where editing existing TeX will cause its position to change
+    (#4415, #4430).
+  * Fixed a memory leak (#4088)
+* PDF support
+  * Added "Xournal++" as the "Creator" in exported PDF metadata (#3523).
+  * Added button to propose replacement background PDF if it is missing and a
+    file with a similar name is found in the same folder as the `.xopp` file
+    (#4165)
+  * Minor performance improvements (#4004).
+* Floating toolbox
+  * Added capability to open the floating toolbox with a secondary button
+    (#3270).
+* Packaging
+  * Added build flags to force or disable X11 linking (#4146).
+  * AppImage: added support for AppImageUpdate (#1915, #4265).
+* Miscellaneous
+  * Improved file size of documents with large amonuts of stroke data by up to
+    15% (#3831, #4065).
```

Comment:
That looks like the wrong issue. I didn't find out which one was meant. Also there is a typo.
```suggestion
  * Improved file size of documents with large amounts of stroke data by up to
    15% (#4065).
```

Reply:
You're right, #3831 should go on the next line.